mozjs/build.rs

```diff
@@ -25,9 +25,8 @@ const ENV_VARS: &'static [&'static str] = &[
     "CXX",
     "CXXFLAGS",
     "MAKE",
-    "MOZ_TOOLS",
+    "MOZ_TOOLS_PATH",
```

Comment:
```suggestion
    "MOZTOOLS_PATH",
```
This would make it consistent with the name `moztools`.